pdpserver/server/log_mgr.go

```diff
@@ -0,0 +1,19 @@
+package server
+
+import (
+	"github.com/Sirupsen/logrus"
+)
+
+type LevelCheckable interface {
+	GetLevel() logrus.Level
+}
+
+func GetLogLevel(logger logrus.FieldLogger) logrus.Level {
+	if lgr, ok := logger.(*logrus.Logger); ok {
+		return lgr.Level
+	} else if lgr, ok := logger.(LevelCheckable); ok {
+		return lgr.GetLevel()
+	}
+	// else
+	return 0 // return the lowest verbosity to be safe
```

Comment:
As for me it's better to use predefined constant instead of plain number. I'd put directly `logrus.PanicLevel`. But it even more readable to define constant here.

```go
const lowestVerbosityLevel = logrus.PanicLevel

func GetLogLevel(logger logrus.FieldLogger) logrus.Level {
	...

	return lowestVerbosityLevel
}